streaming/base/storage/upload.py

```diff
@@ -57,6 +57,7 @@ def get(cls,
             out: Union[str, Tuple[str, str]],
             keep_local: bool = False,
             progress_bar: bool = False,
+            exist_ok: bool = False,
```

Comment:
Can you put `exist_ok` as the last argument ? Otherwise, [this](https://github.com/mosaicml/streaming/blob/main/streaming/base/format/base/writer.py#L119) line will fail.

Reply:
Thanks. didn't realize that! changed.